javaparser-core/pom.xml

```diff
@@ -1,22 +1,22 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Convention is not to include it, was only present in one of our current poms.

Typically license doesn't appear either though.